Backend/src/usecase/implementation/UpdateGameInteractor.test.ts

```diff
@@ -16,7 +16,7 @@ beforeEach(() => {
 })
 
 describe("UpdateGameInteractor", () => {
```

Comment:
reikia testo kuris patikrintu kad per konverterius suvaikšto duomenys ir grąžini output iš interactor normaliai